docs/ext/web.rst

```diff
@@ -30,6 +30,24 @@ To install, run::
     pip install Mopidy-API-Explorer
 
 
+Mopidy-Auto
+===========
+
+https://github.com/gotling/mopidy-auto
+
+Mopidy extension to automate music playback based on time of day.
+
+**Notice!** This extension reacts to the events *tracklist_changed*, *track_playback_ended* and *track_playback_resumed* to accomplish this so other web extensions will not work as expected when this extension is installed.
```

Comment:
Use Sphinx `warning` directive, wrap the lines at less than 80 chars and use double backticks to markup the event names:

```
.. warning::
    This extension reacts to the events ``tracklist_changed``, ``track_playback_ended``, and
    ``track_playback_resumed`` to accomplish its goals. Other web extensions will not work
    as expected when this extension is installed.
```